test/src/e2e_vm_tests/test_programs/should_pass/language/const_decl_with_call_path/src/main.sw

```diff
@@ -7,6 +7,6 @@ fn main() -> u64 {
     let zero = std::constants::ZERO_B256;
     let base_asset_id = std::constants::BASE_ASSET_ID;
     assert(zero == 0x0000000000000000000000000000000000000000000000000000000000000000);
-    assert(base_asset_id == zero);
+    assert(base_asset_id.value == zero);
```

Comment:
```suggestion
    assert(base_asset_id.into() == zero);
```

Reply:
I havent implemented a From<b256> for AssetId on purpose, since I dont want the misunderstanding that the ContractId is still a valid AssetId

Reply:
Huh? This would state that you can turn it into a `b256` and not a `ContractId`. I don't see a misunderstanding. You wouldn't implement an `into` for the `ContractId`.